docs/install.md

```diff
@@ -8,17 +8,20 @@ If you run into any problems with the instructions below, check out [the Trouble
 
 	We recommend that you follow the step to install the `vagrant-hostsupdater` plugin, because if you don't then
 	you will need to manually add the hostnames for each WME site to your local hosts file.
-
-1. Clone this repository as a subdirectory of VVV's `www` folder, and name the folder `wordpress-meta-environment`
-	1. `cd vagrant-local/www` (where `vagrant-local` is the name of your VVV folder)
-	1. `git clone https://github.com/WordPress/meta-environment.git wordpress-meta-environment`
-
-	**It's important that you clone the repository to a folder named `wordpress-meta-environment`.** If you choose
-	a different name, then WME will break during the provision stage, unless you also manually edit the Nginx
-	configuration files for each site to reflect the name you chose.
+	
+	Make sure you have the latest version of VVV.
+
+1. In the `vagrant-local` folder edit `vvv-config.yml` and uncomment the following line:
+       
+       ```yml
+       #wordpress-meta-environment: https://github.com/WordPress/meta-environment.git
+       ```
+       to read
+       ```yml
+       wordpress-meta-environment: https://github.com/WordPress/meta-environment.git
+       ```
```

Comment:
@LoreleiAurora I thought both were required, i.e. the `repo:` value relates to the actual repo URL that gets cloned, so, if it is not uncommented then it will still clone the `https://github.com/WordPress/meta-environment.git` repo anyway?